### DISCUSSION TITLE
Show campsite type

### DISCUSSION THREAD

**Discussion by ERYpTION** [osmandapp/OsmAnd#21322] [Ideas]
Looking at the POI-details for a campsite (tourism=camp_site) I can see additional tagging such as backcountry=*, caravans=*, tents=*, etc.

 I think it would be also be useful to show the value of the [camp_site](https://wiki.openstreetmap.org/wiki/Key:camp_site) key when this has been used to further describe a campsite (basic/standard/serviced/deluxe). 

As an added bonus, it would then also be great if you could search in Osmand and only see campsites of a certain type (as far as I can tell this is not possible now).

Thank you.

**Comment by dmpr0** [comment]
Hi! I have submitted a request at https://github.com/osmandapp/OsmAnd/issues/21332

**Reply by ERYpTION** [reply]
I would just like to add that soon the tag camp_site will be added to the iD preset for tourism=camp_site: https://github.com/openstreetmap/id-tagging-schema/pull/1286
Hopefully this will increase its use.

**Comment by ERYpTION** [comment]
Now that it is possible to see camp site type campsite (basic/standard/serviced/deluxe), is it also possible to filter / search for a specific type? I have made a custom button to show camp sites but would like this to only show basic and standard camp sites. I can't figure out if this can be done?

**Reply by DmitryAlexei** [reply]
Have you tried to create a custom POI search category? https://osmand.net/docs/user/search/search-poi#custom-poi-search

**Reply by ERYpTION** [reply]
I have tried but can only figure out how to filter to camp site, not to the camp site types.

**Comment by DmitryAlexei** [comment]
doesn't it work for you?
<table style="width:100%">
  <tr>
<th><img src="https://github.com/user-attachments/assets/0ffd62af-6a44-426c-9960-3adfa3b3169d"  width="300" height="auto"/> </td>
  </tr>
</table>

**Reply by ERYpTION** [reply]
Yes now it does! Forgot how to create a custom filter. Thanks.
